### ISSUE TITLE
How to run easyrp on Linux? (Debian 8)

### ISSUE THREAD

**Issue by ByLeonTV** [Pizzabelly/EasyRP#105]
Hey,

I am asking this because every thread that I read had no clear solution in it on how to run easyrp on Linux. So if someone would take the time and explain it to me I would be so thankful! Because when I try to execute it with the ./easyrp command it says something with "File or directory not found". Wine doesn't work for me!

Greetings Leon

**Comment by sunsetsonwheels** [comment]
You will have to build it manually it seems. I tried running the prebuilt release version and got the same outcome as you.

I have attached my build from master if you want to try it out. You just need to copy `config.ini` from the prebuilt release zip to the same directory where the EasyRP binary is.

[My EasyRP build](https://anonfiles.com/n8Q5I2Fao7/easyrp)

**Comment by aderchox** [comment]
I have the same issue (just saying)

**Comment by develop331** [comment]
Hello! Unfortunately, EasyRP is windows-only, which means that you can't use it on Linux. Sorry!

**Comment by sunsetsonwheels** [comment]
@itwithlyam I don't think so, I have it working on Linux again, see above comment. All to do is build from source, the prebuilt binary for Linux in releases have some problems.

**Comment by onyxiarivera** [comment]
> @itwithlyam I don't think so, I have it working on Linux again, see above comment. All to do is build from source, the prebuilt binary for Linux in releases have some problems.

`./easyrp: ./easyrp: cannot execute binary file`

**Comment by sunsetsonwheels** [comment]
> > @itwithlyam I don't think so, I have it working on Linux again, see above comment. All to do is build from source, the prebuilt binary for Linux in releases have some problems.
> 
> `./easyrp: ./easyrp: cannot execute binary file`

Have you tried my binary, or building from source yourself?

**Comment by onyxiarivera** [comment]
> > > @itwithlyam I don't think so, I have it working on Linux again, see above comment. All to do is build from source, the prebuilt binary for Linux in releases have some problems.
> > 
> > 
> > `./easyrp: ./easyrp: cannot execute binary file`
> 
> Have you tried my binary, or building from source yourself?

Certainly did sir, to no avail.

**Comment by sunsetsonwheels** [comment]
> > > > @itwithlyam I don't think so, I have it working on Linux again, see above comment. All to do is build from source, the prebuilt binary for Linux in releases have some problems.
> > > 
> > > 
> > > `./easyrp: ./easyrp: cannot execute binary file`
> > 
> > 
> > Have you tried my binary, or building from source yourself?
> 
> Certainly did sir, to no avail.

That's unfortunate, I guess the only way is building it yourself I guess.
